Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Builder
+FROM node:6-slim as builder
+
+RUN apt-get update \
+ && apt-get install -y git python build-essential libicu-dev
+
+RUN git clone https://github.com/matrix-org/freebindfree.git \
+ && cd freebindfree \
+ && make
+
+COPY ./package.json ./package.json
+RUN npm install
+
+# App
+FROM node:6-slim
```

Comment:
Still not sure why we're targetting node 6? Node 10 is definitely going to yield better perf